app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,70 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  # create ServiceTemplate and supporting ServiceResources and ResourceActions
+  # options
+  #   :name
+  #   :description
+  #   :service_template_catalog_id
+  #   :config_info
+  #     :provision
+  #       :dialog_id (or)
+  #       :container_template_id
+  #
+  def self.create_catalog_item(options)
+    options     = options.merge(:service_type => 'atomic', :prov_type => 'generic_container_template')
+    config_info = validate_config_info(options[:config_info])
+    enhanced_config = config_info.deep_merge(
+      :provision => {
+        :configuration_template => ContainerTemplate.find(config_info.fetch_path(:provision, :container_template_id))
```

Comment:
@gmcculloug When we developed Orchestration service there is no configuration_template in resource_action table, so the orchestration_template is added as a resource of the service template. 
I think we now can use configuration_template field to store the container template, like the way we store job template.